grpc/testing/messages.proto

```diff
@@ -94,6 +97,13 @@ message SimpleResponse {
   // Server ID. This must be unique among different server instances,
   // but the same across all RPC's made to a particular server instance.
   string server_id = 4;
+  // gRPCLB Path. This takes the value of either "fallback" or "backend".
+  // The server must fill in "fallback" if it detects that the RPC reached
+  // the server via the "gRPCLB fallback" path, and "backend" if it detects
+  // that the RPC reached the server via "gRPCLB backend" path (i.e. if it got
+  // the address of this server from the gRPCLB server BalanceLoad RPC). Exactly
+  // how this detection is done is server dependant.
+  string grpclb_path = 5;
```

Comment:
The field name is not self-explanatory and "path" leads me to think of something else. How about "route_type"? You may also consider using enums instead of strings.

Reply:
Good point!

I've renamed from path to route_type and changed to an enum